lib/ansible/modules/crypto/entrust/ecs_certificate.py

```diff
@@ -771,19 +771,15 @@ def request_cert(self, module):
                     if self.full_chain_path and self.cert_details.get('chainCerts'):
                         if self.backup:
                             self.backup_full_chain_file = module.backup_local(self.full_chain_path)
-                        chain_string = ''
-                        for chain_cert in self.cert_details.get('chainCerts'):
-                            chain_string = chain_string + chain_cert + '\n'
+                        chain_string = '\n'.join(self.cert_details.get('chainCerts'))
```

Comment:
Now you don't get a final newline. I hope that's ok.

(Having a final newline is nice: you can simply concatenate PEM files without manual intervention by inserting a newline when needed.)